perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -912,7 +930,10 @@ def Prepare(benchmark_spec):
 
   vms[0].Install('pip')
   vms[0].RemoteCommand('sudo pip install python-gflags==2.0')
-  vms[0].RemoteCommand('sudo pip install azure==1.0.0')
+  azure_version_string = ''
+  if FLAGS.azure_lib_version is not None:
+    azure_version_string = '==%s' % FLAGS.azure_lib_version
+  vms[0].RemoteCommand('sudo pip install azure%s' % azure_version_string)
```

Comment:
It's [above](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/593/files#diff-4dfeec95e03bcd628e8a53dabc7874ccR681).